integration/src/test/scala/org/knora/webapi/testcontainers/SipiTestContainer.scala

```diff
@@ -36,19 +40,36 @@ final case class SipiTestContainer(container: GenericContainer[Nothing]) {
     )
   }
 
-  def host: String = container.getHost
-  def port: Int    = container.getFirstMappedPort
-  def sipiBaseUrl: URL = {
+  val port: Int = container.getFirstMappedPort
+
+  val host: String = SipiTestContainer.localHostAddress
+
+  val sipiBaseUrl: URL = {
     val urlString = s"http://$host:$port"
-    URL.fromString(urlString).getOrElse(throw new IllegalStateException(s"Invalid URL $urlString"))
+    println(s"SIPI URL String: $urlString")
+    val url = URL.decode(urlString).getOrElse(throw new IllegalStateException(s"Invalid URL $urlString"))
+    println(s"SIPI URL: $url")
```

Comment:
These are prints by intention, right?

Reply:
The `SipiIT` failed after upgrading the zio-http `Client`.  In principle these `println` could be removed, but I would like to keep these debug output for now just in case the test fails on a different runner.